src/test/java/dev/snowdrop/release/services/ReleaseFactoryTest.java

```diff
@@ -37,53 +37,24 @@
  */
 @QuarkusTest
 public class ReleaseFactoryTest {
-    
+
     @Inject
     ReleaseFactory factory;
-    
+
     private static InputStream getResourceAsStream(String s) {
         return Thread.currentThread().getContextClassLoader().getResourceAsStream(s);
     }
-    
-    @Test
-    public void missingVersionShouldFail() {
-        try {
-            factory.createFrom(getResourceAsStream("missing-version.yml"), getResourceAsStream("pom.xml"), true);
-            fail("should have failed on missing version");
-        } catch (IllegalArgumentException e) {
-            // expected
-            assertTrue(e.getMessage().contains("missing version"));
-        } catch (Throwable e) {
-            fail(e);
-        }
-    }
-    
+
     @Test
-    public void mismatchedVersionShouldFail() {
-        try {
-            factory.createFrom(getResourceAsStream("mismatched-version.yml"), getResourceAsStream("pom.xml"), true);
-            fail("should have failed on mismatched version");
-        } catch (IllegalArgumentException e) {
-            // expected
-            assertTrue(e.getMessage().contains("release version doesn't match"));
-        } catch (Throwable e) {
-            fail(e);
-        }
+    public void missingVersionShouldNotFail() throws Throwable {
```

Comment:
Why did you change these tests? I understand that these cases shouldn't happen when using the tool as expected, however we cannot rule out that someone tries to process a manually edited release definition and I think that we should keep all the validation in place to account for that situation because we don't want to create product tickets using an invalid release definition.

Reply:
@metacosm i made some changes to the code to always set `version` from the `pom.xml`, so there won't be a case where `version` is mismatched or is missing. https://github.com/snowdrop/issues-manager/pull/97/files#diff-c4475b0eb6572d8755db604107dfd103608fdf89816183921cf113950673c721R113